### ISSUE TITLE
What if MC_ANNOUNCE contains bogus IPs

### ISSUE THREAD

**Issue by LPardue** [GrumpyOldTroll/draft-jholland-quic-multicast#101]
The Source IP and Group IP have certain requirements. If I receive an MC_ANNOUNCE where e.g. the Group IP is not in the multicast IP range, what should I do? 

One option is to say that senders MUST send valid values as per RFC 4607, and that receivers of bad values MUST ignore them, or that they MUST ignore them, or something along those lines. I didn't think much through the ramifications of this, maybe you have more opinion or experience?

**Comment by GrumpyOldTroll** [comment]
Yeah, good point.  The reaction should be specified.  My first idea is that declining a join to such a channel with a new reason like UNUSABLE_ADDRESS might be right.  We don't really have a way to object to an announce, but refusing a join is easy.  (BAD_ADDRESS also might be applicable for addresses that are multicast but reserved, or that are in a range that isn't for SSM.)

**Comment by MaxF12** [comment]
Closed by #147
